hexa/pipelines/graphql/schema.graphql

```diff
@@ -138,6 +138,11 @@ type PipelineRecipient {
   pipeline: Pipeline!
 }
 
+enum PipelineType {
+  DEFAULT 
```

Comment:
You haven't changed the PipelineType to another value. Is it what you wanted to push ?

Reply:
It should be ZipFile.